extensions/publisher-command-center/manifest.json

```diff
@@ -11,6 +11,11 @@
 			"package_file": "requirements.txt"
 		}
 	},
+	"environment": {
+		"python": {
+			"requires": ">=3.8, <4"
```

Comment:
If I understand [pep440](https://peps.python.org/pep-0440/#examples) correctly, `"requires": "~=3.8"` would do this same thing, right?